apps/admin_app/lib/admin_app_web/auth_pipeline/role_helper.ex

```diff
@@ -1,22 +1,44 @@
 defmodule AdminAppWeb.RoleHelper do
-  @moduledoc false
+  @moduledoc """
+  Provides helper functions to work with roles.
+  """
 
-  def load_role_permissions do
-    File.cwd!()
-    |> Path.join("/priv/role_manifest.yaml")
-    |> YamlElixir.read_from_file()
-  end
+  @doc """
+  Returns the access status based on supplied inputs.
+
+  It checks whether the `controller` and it's `action` supplied in
+  input falls under any of the supplied `permission_codes` declared
+  in `role_mainfest.yaml`. In case it does the function returns
+  `true` otherwise `false`.
+
+  In case the "role" passed in the input is "admin", in that
+  case access to all the controller functions is allowed and,
+  the funtion returns `true`.
+  """
+  @spec access_status(String.t(), [String.t()], String.t(), String.t()) :: boolean
```

Comment:
`is_accessible` is a better name (see style guide).
The clause for "admin" is not justified, since the role has been seeded. Put some effort in writing a full blown manifest 😏